pdc/apps/release/models.py

```diff
@@ -371,7 +371,7 @@ class Meta:
         ordering = ("variant", "arch")
 
     def __unicode__(self):
-        return u"%s.%s" % (self.variant, self.arch)
+        return u"%s.%s, deleted: %s" % (self.variant, self.arch, self.deleted)
```

Comment:
I just to want to if a model is changed the it's unicode could reflect that. Because changeset will check if original and modified obj export value is same, if same then will not record the change. And this unicode method will be refered by other model,  like `Variant`.